crates/re_sdk_comms/src/buffered_client.rs

```diff
@@ -191,16 +191,25 @@ fn msg_encode(
                 if let Ok(msg_msg) = msg_msg {
```

Comment:
```suggestion
                if Ok(msg_msg) = msg_msg else {
```

Would a `let-else` here work better to reduce the nesting a bit?

Reply:
good idea!